audius-cli

```diff
@@ -601,33 +602,46 @@ def auto_upgrade(ctx, remove, cron_expression):
 
 @cli.command()
 @click.pass_context
-def update_chainspec_set_genesis(ctx):
+def update_chainspec(ctx):
     """
-    Sets the genesis address in the chainspec with the appropriate
-    keys for sealing blocks from the override.env.
-    This command isonly intended to be run by the genesis validator
-    of the network.
+    Prepares the discovery chainspec and static nodes, pulling from
+    relevant environment variables.
     """
-    # Get signer from override.env
-    override_env = ctx.obj["manifests_path"] / "discovery-provider" / "override.env"
-    override_env_data = dotenv.dotenv_values(override_env)
-    signer = override_env_data.get("audius_delegate_owner_wallet")
-    signer = signer.replace("0x", "")  # trim 0x from pubkey
+    _update_chainspec(ctx)
+
+
+def _update_chainspec(ctx):
```

Comment:
Why a separate underscore function? This does not accomplish anything

Reply:
if it's decorated with click, it can't be called from other functions. got another name preference? or alt structure?

Reply:
Use ctx.invoke

Reply:
nice